.github/workflows/ci_integration.yml

```diff
@@ -292,3 +292,18 @@ jobs:
           tag-latest-for-branch: "${{ steps.config.outputs.tag-latest-for-branch }}"
           tag-latest-for-py: "${{ steps.config.outputs.tag-latest-for-py }}"
           target: "final-dev"
+      - name: "Build and Push `final-dev` arm64 as well, only for next + latest"
+        if:
+          steps.config.outputs.tag-latest-for-branch == 'true' &&
+          steps.config.outputs.branch == 'next'
+        uses: "./.github/actions/build-nautobot-image"
+        with:
+          branch: "${{ steps.config.outputs.branch }}"
+          image: "ghcr.io/nautobot/nautobot-dev"
+          platforms: "linux/arm64"
+          push: "true"
+          python-version: "${{ matrix.python-version }}"
+          tag-latest: "${{ steps.config.outputs.tag-latest }}"
+          tag-latest-for-branch: "${{ steps.config.outputs.tag-latest-for-branch }}"
```

Comment:
I wonder actually if this is going to cause any conflicts with the `latest` tag for amd64 builds above?

Maybe we need more of an if/else case instead, e.g.:

- build arm64 alone if *not* latest + next
- else build arm64 *and* amd64 in the same step